java/piranha/src/main/java/com/uber/piranha/XPFlagCleaner.java

```diff
@@ -881,6 +899,45 @@ private boolean isCheckedXPFlagName(ExpressionTree tree) {
   public Description matchMethod(MethodTree tree, VisitorState state) {
     if (disabled) return Description.NO_MATCH;
 
+    if (!configTestMethodProperties.isEmpty() && tree != null && tree.getBody() != null && tree.getBody().getStatements() != null) {
```

Comment:
@mkr-plse - This entire logic is very specific and I'm not happy with this approach. 
But need your inputs to get this better and generic